dashboard/src/app/utils/time.ts

```diff
@@ -0,0 +1,13 @@
+export function convertSecondsToReadableTime(secondsInput: number): string {
```

Comment:
There's a similar implementation on #1031

Let's sync to reuse the same.

Reply:
lets merge one of the prs and then merge main to the other one to sync up on the function

Reply:
> lets merge one of the prs and then merge main to the other one to sync up on the function

Sure, yours can go first.

Reply:
> > lets merge one of the prs and then merge main to the other one to sync up on the function
> 
> Sure, yours can go first.

Sure can you approve that PR and then we can add unit tests in this PR as well so we don't have 2 prs open.